ddl/placement/rule.go

```diff
@@ -58,7 +58,6 @@ func NewRule(role PeerRoleType, replicas uint64, cnst Constraints) *Rule {
 		Count:          int(replicas),
 		Constraints:    cnst,
 		LocationLabels: []string{"region", "zone", "rack", "host"},
-		IsolationLevel: "region",
```

Comment:
Need to modify some test？

Reply:
I did not find one by grep, i.e. I don't find any other assignment to this field. I remembered that tests will assert on rules only.

Reply:
Addressed. Remind me if you find another test needs to be fixed.